.github/workflows/antithesis-test.yml

```diff
@@ -0,0 +1,112 @@
+---
+name: Build and trigger Antithesis exploration
+
+environment: Antithesis
```

Comment:
Do we need `environment: Antithesis` mentioned 3 times? Would 1 be enough?

Reply:
This first one was a mistake. Kept `environment` for those under jobs for now just to be safe.